vllm/_custom_ops.py

```diff
@@ -1,17 +1,17 @@
 import contextlib
 import functools
 from typing import List, Optional, Tuple, Type
-
 import torch
-
+import habana_frameworks.torch.hpu as hthpu
```

Comment:
This is going to fail in non-hpu dockers

Reply:
ok then may i suggest moving _is_hpu() from setup.py to utils so that we can import it anywhere ?
or do you have an alternate solution ?

Reply:
as vllm._c is needed for build_custom_ops, can we just check https://github.com/HabanaAI/vllm-fork/blob/habana_main/setup.py#L291, then import vllm._c?

or if we want to import hthpu, then we have to use try except

Reply:
its already doing that in setup.py
https://github.com/HabanaAI/vllm-fork/blob/habana_main/setup.py#L489 , _build_custom_ops() is for non-hpu only.

anyways i added a try catch for habana api, this should work for non-habana dockers. 
please review.